sdk/core/azure-core/src/main/java/com/azure/core/http/policy/AzureKeyCredentialPolicy.java

```diff
@@ -8,10 +8,34 @@
 import java.util.Objects;
 
 /**
- * Pipeline policy that uses an {@link AzureKeyCredential} to set the authorization key for a request.
- * <p>
- * Requests sent with this pipeline policy are required to use {@code HTTPS}. If the request isn't using {@code HTTPS}
- * an exception will be thrown to prevent leaking the key.
+ * The {@code AzureKeyCredentialPolicy} class is an implementation of the {@link KeyCredentialPolicy} interface. This
+ * policy uses an {@link AzureKeyCredential} to set the authorization key for a request.
+ *
+ * <p>This class is useful when you need to authorize requests with a key from Azure. It ensures that the requests are
+ * sent over HTTPS to prevent the key from being leaked.</p>
+ *
+ * <p>Requests sent with this pipeline policy are required to use {@code HTTPS}. If the request isn't using
+ * {@code HTTPS} an exception will be thrown to prevent leaking the key.</p>
```

Comment:
This is duplicative. 
```suggestion
 * <p>This class is useful when you need to authorize requests with a key from Azure.</p>
 *
 * <p>Requests sent with this pipeline policy are required to use {@code HTTPS}. If the request isn't using
 * {@code HTTPS} an exception will be thrown to prevent leaking the key.</p>
```